CHANGELOG.md

```diff
@@ -59,11 +61,15 @@ accessed directory. examples/cookbook/recipe_coalesce_pool.cpp shows how to do
 this.
 
 ### Fixed
-- Bug in ResourceManager::copy/memset when given a pointer offset into an allocation.
+
+- Bug in ResourceManager::copy/memset when given a pointer offset into an
+allocation.
 
 - Memory leak in judyL2Array.
 
-- While replay already was recording release operations, the tool was not actually replaying them.  A fix was implemented so that the replay tool will now also replay any captured release operations.
+- While replay already was recording release operations, the tool was not
+actually replaying them.  A fix was implemented so that the replay tool will
```

Comment:
What caused these changes?

Reply:
I line wrapped stuff correctly.

Reply:
Hmm.  I assumed that since this was markdown that line wrapping was taken care of.  It seems to render fine in either case.

Reply:
What is the "correct" line wrap?  80 columns?  I'll try to remember that for the next time.